controllers/storagecluster/cephfilesystem.go

```diff
@@ -153,6 +229,11 @@ func (obj *ocsCephFilesystems) ensureDeleted(r *StorageClusterReconciler, sc *oc
 				return reconcile.Result{}, fmt.Errorf("uninstall: Failed to delete CephFileSystem %v: %v", foundCephFilesystem.Name, err)
 			}
 		}
+		// delete csi subvolume group for particular filesystem
+		err = r.deleteDefaultSubvolumeGroup(cephFilesystem.Name, cephFilesystem.Namespace, cephFilesystem.ObjectMeta.OwnerReferences)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
```

Comment:
SVG needs to be deleted before cephFileSystem, right?
So, we should move this to line:224, i.e, before the Deletion of the filesystem.

Reply:
IMO it wont be a problem but still moved to avoid any race condition